### PR TITLE
Added 3 tests (workflows)

### PR DESCRIPTION
**Pull request by nastya-kalashnikova**

### CODE REVIEW

File: .github/workflows/FibonacciLFSRGenerator.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        default: '1000000'
+        type: choice
+        options: ['1', '10', '1000000']
+      stream_number:
+        description: >
+          Bitstreams (number of Sequences)
+          NOTE: For Significance level = 0.001, a sample should have at least 1000 sequences!
+          The sample should be on the order of the inverse of the significance level (α^-1).
+        required: true
+        default: '100'
```
Comment (AbhayCharan) [inline .github/workflows/FibonacciLFSRGenerator.yml]: please use 1000 bitstreams as default value

**Review by AbhayCharan — CHANGES_REQUESTED** [review]

File: .github/workflows/GaloisLeftLFSRGenerator.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        default: '1000000'
+        type: choice
+        options: ['1', '10', '1000000']
+      stream_number:
+        description: >
+          Bitstreams (number of Sequences)
+          NOTE: For Significance level = 0.001, a sample should have at least 1000 sequences!
+          The sample should be on the order of the inverse of the significance level (α^-1).
+        required: true
+        default: '100'
```
Comment (AbhayCharan) [inline .github/workflows/GaloisLeftLFSRGenerator.yml]: this one too

File: .github/workflows/FibonacciLFSRGenerator.yml (hunk 1 above)
Comment (nastya-kalashnikova) [reply .github/workflows/FibonacciLFSRGenerator.yml]: okay done

**Review by AbhayCharan — CHANGES_REQUESTED** [review]
test Request changes

File: .github/workflows/GaloisLeftLFSRGenerator.yml (hunk 2 above)
Comment (nastya-kalashnikova) [reply .github/workflows/GaloisLeftLFSRGenerator.yml]: done!

**Review by AbhayCharan — APPROVED** [review]

**Pull request merged**